gcloud-java-dns/src/main/java/com/google/gcloud/dns/ManagedZoneInfo.java

```diff
@@ -0,0 +1,334 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.gcloud.dns;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.google.common.base.MoreObjects;
+import com.google.common.collect.ImmutableList;
+
+import org.joda.time.DateTime;
+import org.joda.time.format.ISODateTimeFormat;
+
+import java.io.Serializable;
+import java.math.BigInteger;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * This class is a container for the managed zone metainformation. Managed zone is a resource that
```

Comment:
Personally I prefer the public [doc](https://cloud.google.com/dns/api/v1/managedZones) but reversed.

 A ManagedZone is a resource that represents a DNS zone hosted by the Cloud DNS service.
A zone is a subtree of the DNS namespace under one administrative responsibility.

I would not use metainformation as one word and I am not sure that is correct (managed zone) is meta information about a DNS zone.